pkg/services/auth/authimpl/token_cleanup.go

```diff
@@ -7,12 +7,16 @@ import (
 	"github.com/grafana/grafana/pkg/infra/db"
 )
 
+const (
+	ExpiredTokenCleanupInterval = time.Hour * 12
```

Comment:
extracts this to be exported to revoking only up to this hour

Reply:
Why do we need to export this one?

Reply:
This is being used in the enterprise repo, for querying tokens based a period back.

could be that we just configured that separately.

```go
// it's being used as part of the `removeOldUserRevokedTokens`
err := s.removeOldUserRevokedTokens(ctx, user, authimpl.ExpiredTokenCleanupInterval, maxConcurrentUserSessions)
```

Reply:
we do not need to export it